package.json

```diff
@@ -27,7 +27,7 @@
     "clean": "hardhat clean && rm -rf artifacts && rm -rf typechain-types && rm -rf cache && rm -rf coverage && rm -f coverage.json && rm -f network.js* && rm -f network.d*",
     "up": "npm install && npx hardhat compile && npm run build && hardhat node",
     "test": "npm run test:unit && npm run test:e2e",
-    "test:unit": "hardhat coverage --testfiles \"test/unit\" && istanbul check-coverage ./coverage.json --statements 100 --branches 96 --functions 100 --lines 100",
+    "test:unit": "hardhat coverage --testfiles \"test/unit\" && istanbul check-coverage ./coverage.json --statements 100 --branches 98 --functions 100 --lines 100",
```

Comment:
Nice!  There's some added `nonReentrant` tests added in the "runSQLs" pull request, and once those land we should be up to 100% coverage.